targets/XPlatCppSdk/source/code/source/playfab/PlayFabIOSHttpPlugin.mm

```diff
@@ -0,0 +1,346 @@
+#include <stdafx.h>
+#import <UIKit/UIKit.h>
+
+#include <playfab/PlayFabIOSHttpPlugin.h>
+#include <playfab/PlayFabSettings.h>
+#include <stdexcept>
+#include <vector>
+
+namespace PlayFab
+{
+    struct PlayFabIOSHttpPlugin::RequestImpl
+    {
+        RequestImpl() :
+            sessionTask(nil)
+        {
+
+        }
+
+        virtual ~RequestImpl()
+        {
+            sessionTask = nil;
+        }
+
+        void Cancel()
+        {
+            if(sessionTask)
+            {
+                [sessionTask cancel];
+            }
+        }
+
+        __strong NSURLSessionUploadTask* sessionTask;
+    };
+
+}
```

Comment:
can you not include whole code in same { }? sorry for my lack of understanding of .mm file